docs/proposals/formatted-output.md

```diff
@@ -332,57 +334,234 @@ $REGISTRY/$REPO@sha256:a3785f78ab8547ae2710c89e627783cfa7ee7824d3468cae6835c9f4e
     └── sha256:50fd0dc107d84b5e7b402688000a7ed3aaf8a2692d5cb74da5277fa3c4cecf15
 ```
 
-View an artifact's referrers manifest in pretty JSON output. The following fields should be outputted:
+> [!NOTE]
+> The `--format` flag replaces the existing `--output` flag. The `--output` has been marked as "deprecated" in ORAS v1.2.0 and will be removed in future releases.
```

Comment:
Currently, the `--output` flag is still visible in the help doc. I think we can just hide it. Should we mention this in the spec?

Reply:
Or we could consider moving this discussion to an issue and create a doc to elaborate on the feature lifecycle management.

Reply:
@wangxiaoxuan273 @Wwwsylvia @oras-project/oras-cli I created an issue to track this discussion https://github.com/oras-project/oras-www/issues/460. We can discuss it in the issue and align on the deprecation policy, then make the changes to corresponding deprecated flags. I will mark this comment as resolved.